sdk/tests/iot/hub/test_az_iot_hub_client_commands.c

```diff
@@ -0,0 +1,585 @@
+#endif
+
+/*static int teardown(void **state) {
+     (void)state;
+     printf("*** in teardown ***\n");
+     //az_log_set_message_callback(NULL);
+     return 0;
+}*/
+
+int test_az_iot_hub_client_commands()
+{
+#ifndef AZ_NO_PRECONDITION_CHECKING
+  SETUP_PRECONDITION_CHECK_TESTS();
+#endif // AZ_NO_PRECONDITION_CHECKING
+
+  const struct CMUnitTest tests[] = {
+#ifndef AZ_NO_PRECONDITION_CHECKING
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_NULL_out_topic_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_response_get_publish_topic_NULL_client_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_zero_size_buffer_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_EMPTY_request_id_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_AZ_SPAN_NULL_request_id_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_NULL_client_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_parse_received_topic_EMPTY_received_topic_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_parse_received_topic_AZ_SPAN_NULL_received_topic_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_NULL_out_request_fail),
+#endif // AZ_NO_PRECONDITION_CHECKING
+    cmocka_unit_test(test_az_iot_hub_client_commands_response_get_publish_topic_succeed),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_user_status_succeed),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_user_status_small_buf_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_INSUFFICIENT_BUFFER_for_prefix_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_INSUFFICIENT_BUFFER_for_status_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_response_get_publish_topic_INSUFFICIENT_BUFFER_for_reqid_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_with_component_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_c2d_topic_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_get_property_topic_fail),
+    cmocka_unit_test(
+        test_az_iot_hub_client_commands_parse_received_topic_property_patch_topic_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_topic_filter_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_parse_received_topic_response_topic_fail),
+    cmocka_unit_test(test_az_iot_hub_client_commands_logging_succeed)
+  };
+
+  return cmocka_run_group_tests_name("az_iot_hub_methods", tests, NULL, NULL);
```

Comment:
?
```suggestion
  return cmocka_run_group_tests_name("az_iot_hub_commands", tests, NULL, NULL);
```

Reply:
Yup, will make this on next iteration of this.